README.rst

```diff
@@ -60,34 +60,119 @@ by a number of developers over the years including
+  $ cd django-premis-event-service
+
+Create a virtualenv_ environment
+""""""""""""""""""""""""""""""""
+
+.. _virtualenv: https://virtualenv.pypa.io/en/stable/
+
+.. code-block :: sh
+
+    $ mkvirtualenv premis-event-service # to create and enter the virtualenv
+    (premis-event-service) $ deactivate # to exit the virtualenv
+    $ workon premis-event-service # to reactivate the virtualenv
+
+
+Install the requirements using pip_
+"""""""""""""""""""""""""""""""""""
+
+.. _pip: https://pip.pypa.io/en/stable/
+
+.. code-block :: sh
+
+    (premis-event-service) $ pip install -r requirements.txt # install dependencies from text file
+
+
+Run the tests using tox_
+""""""""""""""""""""""""
+
+.. _tox: https://tox.readthedocs.io/en/latest/
+
+.. code-block :: sh
+
+    (premis-event-service) $ tox
+
+
+Note that the tests will be run in multiple environments, most importantly in distinct environments for Django major versions 1.7-1.10. Tests will also be run against the Django master branch, which is a development branch and prone to failure. These failures are ignored by the PREMIS Event Service testing configuration, and you can likely ignore them as well, particularly if you are using one of the other Django major versions against which the tests should pass.
+
+
+Apply the migrations
+""""""""""""""""""""
+
+.. code-block :: sh
+
+    (premis-event-service) $ python manage.py migrate
+
+
+Start the development server
+""""""""""""""""""""""""""""
+
+.. code-block :: sh
+
+    (premis-event-service) $ python manage.py runserver 9999
+
+
+This will start the development server listening locally on port 9999. You may want to change the port number, passed as the first argument to the ``runserver`` command.
+
+
+View the web UI in a browser
+""""""""""""""""""""""""""""
+
+Navigate to ``http://localhost:9999`` (or whatever port you chose) to see the UI of the app.
+
+
+Developing Using Docker and MySQL as a Backend
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Install Docker_
+"""""""""""""""
 
 .. _Docker: https://docs.docker.com
 
+On Debian-derived Linux distros, you can use ``apt-get`` to install. If you're on a different OS, check the Docker site for instructions.
+
+
 Install Docker Compose
+""""""""""""""""""""""
 
 .. code-block :: sh
 
   $ pip install docker-compose
 
+Alternatively, you may want to install ``docker-compose`` using your system's package manager.
 
-Clone the repository.
+
+Clone the repository
+""""""""""""""""""""
 
 .. code-block :: sh
 
-  $ git clone https://github.com/unt-libraries/django-premis-event-service.git
+  $ git clone https://github.com/unt-libraries/django-premis-event-service.git # check the repo for the latest official release if you don't want the development version at HEAD on the master branch
   $ cd django-premis-event-service
 
 
-Start the app and run the migrations.
+Starting the app
```

Comment:
What do you think about adding a `Navigate to ``http://localhost:8000/event/`` to see the UI of the app` similar to what is in the developing natively info either in this `Starting the app` section or under its own subheading like you have it for native deployment?